vaxrank/report.py

```diff
@@ -71,8 +71,8 @@ def ascii_report_from_ranked_vaccine_peptides(
         for j, vaccine_peptide in enumerate(vaccine_peptides):
             mutant_protein_fragment = vaccine_peptide.mutant_protein_fragment
             amino_acids = mutant_protein_fragment.amino_acids
-            mutation_start = mutant_protein_fragment.mutant_amino_acid_start_offset
```

Comment:
Probably better for @iskandr to check this out, but was the problem that the "underlined" AA was one too far to the right? In which case this moves the underline one *more* to the right? I'm pretty unfamiliar with the offsets used around all of this, though...